app/javascript/packages/phone-input/index.spec.ts

```diff
@@ -110,7 +111,7 @@ describe('PhoneInput', () => {
     expect(phoneNumber.validity.valueMissing).to.be.true();
 
     await userEvent.type(phoneNumber, '5');
-    expect(phoneNumber.validationMessage).to.equal('Phone number is not valid');
+    expect(phoneNumber.validationMessage).to.equal('Enter a 10 digit phone number.');
```

Comment:
It might be good to create a separate test case to check for both US and international validation, similar to what you had before with the revisions "it validates input for US numbers".

We're testing this indirectly with the test case below ("with single option, it validates phone from region"), but could be good to have the standalone test for the non-singular context.

Basically copy this test case and have two versions:

- `it('validates input for U.S. number')`
- `it('validates input for international number')`

Or:

```ts
context('with international number', () => {
  it('validates input', () => {
    // ...
  });
});
```